.gitattributes

```diff
@@ -0,0 +1,45 @@
+# Automatically normalize line endings.
+* text=auto
+
+# Files and directories to exclude from the generated archive (export-ignore).
+# These will be omitted from GitHub/Packagist ZIP archives to keep packages small.
+
+# Tests and test fixtures
+/tests/ export-ignore
+/tests/** export-ignore
```

Comment:
I think the line above will cover for the second (aka I think we can remove this second line of `/tests/** export-ignore`)